README.md

```diff
@@ -1,8 +1,8 @@
 # To run the app, follow these steps:
 
 1. Run 'go setup'.
-2. Open two terminal windows.
-3. Run 'go client' in one of the windows, and 'go server' in the other.
-4. Open localhost:8081 and start developing
+2. Open three terminal windows.
```

Comment:
yeah i'm thinking of just running mongo in the background...so many windows right now.